.github/workflows/pull-request-title.yml

```diff
@@ -0,0 +1,22 @@
+name: PR Title
```

Comment:
It's unclear if this should be in `/.github/` directory or the parent.
How does this special repo `.github` apply its contents?
I didn't find any docs from a scan online. (Of course, its not an easy search)

This file does work, for this repo:
https://github.com/ClipboardHealth/.github/actions/runs/7509833906/job/20447464365?pr=3

I suspect though, that we want to move this file from `/.github/workflows/` to `/workflows/`

Reply:
Ok, I'll move it. I'll also add a readme.